oeps/best-practices/oep-0066-bp-authorization.rst

```diff
@@ -405,27 +405,70 @@ in the CMS.
   v1 libraries (deprecated) granted access to libraries on a course by course basis 
   and was controlled by student_courseaccessrole.
 
-student/learner
+Student/Learner
 ---------------
 student/learner is an implicit role. 
 
 It is not currently controlled by a system/protocol 
 whose primary focus is authorization.  
 
-system-wide roles
-----------------------
+System-wide Roles
+-----------------
 System-wide roles are configurable and can differ between different Open edX instances. This 
 means that different instances can have different system-wide roles. 
 
 System-wide user-roles (user assignments to a specific system-wide role) are
 stored in a central user service (currently LMS) and communicated via JWT Tokens. 
 
-Each feature/service enforces the roles in its own codebase,
-if the feature/service is using the system-wide role.
+If a feature/service utilizes a system-wide role, the feature/service enforces the roles in its own codebase.
+
+Example System-wide Role:
 
-Example system-wide role:
 * Global Staff - propogated in JWTs as the "administrator" field
 
+Example AuthZ User Access Flows
+-------------------------------
+.. image:: oep-0066/Open_edX_Authorization_User_Flows.png
```

Comment:
Can you provide a link to the diagram source as well?
```
..
   Something like this might add a hidden note in the doc.
```

Reply:
The diagram was created specifically for this OEP. 

It is based on one in https://openedx.atlassian.net/wiki/spaces/AC/pages/935919751/Authorization, but with a few modifications aimed at highlighting certain use cases. The changes are significant enough that I don't think we'd want to link to that diagram as the source.

Reply:
Right. I meant the source of the new png. How was it created, and what happens if someone wants to update it? (Same might be true of other diagrams.)

[From OEP-19](https://open-edx-proposals.readthedocs.io/en/latest/best-practices/oep-0019-bp-developer-documentation.html):
> Exported Diagrams. In order to keep diagrams updated across time, they will need to be exported from whichever tool they were created in, with the exported version maintained in GitHub.
> 
> - draw.io diagrams will be exported and imported as XML files.
> - Lucidchart diagrams will be exported and imported as Visio (VDX) files.